assets/xml/scenes/overworld/spot01_pal_n64.xml

```diff
@@ -2,6 +2,11 @@
     <File Name="spot01_scene" Segment="2">
         <Path Name="spot01_scenePathList_0003D0" Offset="0x03D0" NumPaths="3"/>
         <Cutscene Name="gKakarikoVillageIntroCs" Offset="0xA560"/>
+        <Cutscene Name="gKakarikoVillageRavagedCs" Offset="0x4A50"/>
+        <Cutscene Name="gKakarikoVillageNocturneOfShadowCs" Offset="0x6DF4"/>
+        <Cutscene Name="gKakarikoVillageCreditsCs" Offset="0x8EF4"/>
+        <Cutscene Name="gKakarikoVillageWellEmptyingCs" Offset="0x80A0"/>
```

Comment:
I think its more commonly referred to as "draining" the well, but this is also fine